framework/src/org/checkerframework/common/value/ValueVisitor.java

```diff
@@ -129,6 +134,20 @@ public Void visitAnnotation(AnnotationTree node, Void p) {
     }
 
     @Override
+    protected void commonAssignmentCheck(
+            AnnotatedTypeMirror varType,
+            ExpressionTree valueExp,
+            /*@CompilerMessageKey*/ String errorKey) {
+
+        // This is definitely a hack. However, it does prevent some spurious warnings
+        // from the Value Checker when interacting with unannotated bytecode.
+        if (varType.toString().contains("@")
```

Comment:
Just `varType.toString().contains("@")` does cause a bunch of tests to fail, but adding the second condition causes them not to. I don't actually think this code is correct; I just wanted some help understanding what's going on here.